docs/src/main/sphinx/release/release-456.md

```diff
@@ -0,0 +1,86 @@
+# Release 456 (5 Sep 2024)
+
+## General
+
+* Expose additional JMX metrics about resource groups, including CPU and memory
+  usage, limits, and scheduling policy. ({issue}`22957`)
+* Add [adaptive plan optimization](/optimizer/adaptive-plan-optimizations) which
+  can reorder joins at runtime. The feature can be disabled with the
```

Comment:
```suggestion
* Add [adaptive plan optimization](/optimizer/adaptive-plan-optimizations) which
  can reorder joins at runtime when FTE is used. The feature can be disabled with the
```

Only works with FTE it seems. The name of the property implies that but good to be explicit.

Reply:
The docs have that information already and are linked right above.. I just wanted to keep it shorter. Wdyt @martint ?

Reply:
I would also rephrase it in terms of user-visible impact:

> Improve performance of queries involving joins when fault tolerant execution is enabled. This optimization can be disabled via the xxx configuration ...


(eventually, we're going to make this optimizations always-on and remove the fallback)